src/sidebar/FeedList/dragdrop/useDragDropNode.ts

```diff
@@ -1,11 +1,12 @@
-import { useContext, useState } from 'react';
+import { useContext, useMemo, useState } from 'react';
 import { useDispatch } from 'react-redux';
 
 import { InsertMode, NodeMeta, NodeType } from '../../../model/feeds';
 import { useAppSelector } from '../../../store/hooks';
-import feedsSlice, { selectDescendentNodeIds } from '../../../store/slices/feeds';
+import feedsSlice, { makeselectDescendentNodeIds } from '../../../store/slices/feeds';
 import { UnreachableCaseError } from '../../../utils/UnreachableCaseError';
 import { RelativeDragDropPosition, relativeDragDropPosition } from '../../../utils/dragdrop';
+import useNamedCallback from '../../../utils/hooks/useNamedCallback';
```

Comment:
this should only be used for debugging

Reply:
We leave it for now, but .. https://github.com/dermeck/feeds-sidebar/pull/176/commits/62211ac44a074c33f023bd63f11f2b02ee9da3e4